composer.json

```
@@ -31,7 +31,7 @@
         }
     },
     "require-dev": {
-        "composer/composer": "^1.10.19",
+        "composer/composer": "^1.10|^2.0",
```

I think it's worth keeping the patch version for 1.x, as this will be used in `--prefer-lowest`. We may as well bump it to the latest 1.x which [fixes a command injection vulnerability on Windows](https://github.com/composer/composer/releases/tag/1.10.23). 👍🏻
```suggestion
        "composer/composer": "^1.10.23|^2.0",
```